core-bundle/contao/library/Contao/Controller.php

```diff
@@ -593,6 +593,11 @@ public static function getContentElement($intId, $strColumn='main')
 		}
 		else
 		{
+			if ($contentElementReference?->attributes['classes'] ?? null)
+			{
+				$objRow->classes = $contentElementReference->attributes['classes'];
```

Comment:
Don’t we have to clone or unregister it from the registry before making modifications?

Reply:
Should we also check if `$objRow->classes` is already set and in that case merge the two?

Reply:
I thought about that too, but the current content element implementation does not do that either, right?

Reply:
I’m not sure what you mean by “current content element implementation”, can you post a link to the code?

Reply:
it's in the initial text. https://github.com/contao/contao/blob/5.x/core-bundle/contao/elements/ContentElement.php#L260

Reply:
No modifications to the model are made there and the classes are merged there too so I don’t understand what you mean by “current content element implementation does not do that either”?
To which of my two comments does your question refer?